File: model/genre.go
package model

import (
	"github.com/jinzhu/gorm"
)

type Genre struct {
	gorm.Model
	GenreID    int64  `gorm: "column: genre_id; type: bigint; primary_key; unique; not null"`
	GenreTitle string `gorm: "column: genre_title; type: nvarchar(50); unique; not null"`
}

File: admin/backend/src/db/model/genre.ts
import { Model, snakeCaseMappers } from 'objection';

export class Genre extends Model {
    genreId: string;
    genreTitle: string;
    
    static get tableName() {
        return 'genres';
    }

    static get idColumn() {
        return 'genre_id';
    }

    static get columnNameMappers() {
        return snakeCaseMappers();
    }
}

export default Genre;